etc/profile-a-l/electron.profile

```diff
@@ -16,7 +16,7 @@ include disable-xdg.inc
 
 whitelist ${DOWNLOADS}
 whitelist ${HOME}/.config/Electron
-whitelist ${HOME}/.config/electron-flag*.conf
+whitelist ${HOME}/.config/electron[0-9]*-flag*.conf
```

Comment:
This will break all program which `ignore` the "old" line.

Update

```console
$ grep 'ignore whitelist ${HOME}/.config/electron-flag\*.conf' /etc/firejail/*
/etc/firejail/atom.profile:ignore whitelist ${HOME}/.config/electron-flag*.conf
/etc/firejail/code.profile:ignore whitelist ${HOME}/.config/electron-flag*.conf
/etc/firejail/github-desktop.profile:ignore whitelist ${HOME}/.config/electron-flag*.conf
/etc/firejail/notable.profile:ignore whitelist ${HOME}/.config/electron-flag*.conf
```